### PR TITLE
added-progress-bar-file

### PR DESCRIPTION
**Pull request by murtazshaikh**
review pr...

### CODE REVIEW

**Review by OSAMA478 — APPROVED** [review]
nice work.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    content: '';
+    position: absolute;
+    width: 16rem;
+    height: 6px;
+    background: #b5b5b58c;
+    border-radius: 12px;
+}
+
+@keyframes animate {
+    100%{
```
Comment (syedabdullahali) [inline style.css:93]: ? 100%

**Review by syedabdullahali — APPROVED** [review]

**Review by anask601 — APPROVED** [review]

**Review by syedabdullahali — APPROVED** [review]

File: style.css (hunk 1 above)
Comment (nehalskdev) [reply style.css:93]: @murtazshaikh how exactly this working?

Comment (murtazshaikh) [reply style.css:93]: we are directly reaching to our end point by specifying 100% .

**Pull request merged**